fiftyone/utils/labels.py

```
@@ -1198,3 +1206,223 @@
+    if any(v is None for v in [det.location, det.rotation, det.dimensions]):
+        logger.warning(
+            f"Skipping detection (id={det.id}) in sample {sample.id} "
+            "because location, rotation, or dimensions is missing"
+        )
+        return None
+
+    points_cam, rotation_cam = fou3d.multiple_coordinate_transform(
+        det.location, det.rotation, transforms, forward_flags
+    )
+    if getattr(det, "instance", None) is None:
+        added_instance = True
```

Ruff (0.12.2)

Local variable `added_instance` is assigned to but never used

Remove assignment to unused variable `added_instance`

(F841)